src/ReactiveDomain.Tools/Shovel/Bootstrap.cs

```diff
@@ -0,0 +1,118 @@
+﻿using EventStore.ClientAPI;
+using EventStore.ClientAPI.SystemData;
+using System;
+using System.Configuration;
+using System.IO;
+using System.Net;
+using System.Reflection;
+
+namespace Shovel
+{
+    public class Bootstrap
+    {
+        private EventShovelConfig _eventShovelConfig = new EventShovelConfig();
+        private Assembly _transformerAssembly;
+
+        public bool Loaded { get; private set; }
+
+        public void Load()
+        {
+            _eventShovelConfig.SourceConnection =
+                ConnectToEventStore(ReadSetting("sourceTcpAddress"), int.Parse(ReadSetting("sourcePort")));
+            _eventShovelConfig.TargetConnection =
+                ConnectToEventStore(ReadSetting("targetTcpAddress"), int.Parse(ReadSetting("targetPort")));
+            _eventShovelConfig.SourceCredentials =
+                new UserCredentials(ReadSetting("sourceUsername"), ReadSetting("sourcePassword"));
+            _eventShovelConfig.TargetCredentials =
+                new UserCredentials(ReadSetting("targetUsername"), ReadSetting("targetPassword"));
+
+            try
+            {
+                _transformerAssembly = Assembly.LoadFrom(@".\EventTransformer.dll");
+            }
+            catch (FileNotFoundException)
+            {
+                Console.WriteLine("No transformer assembly found.");
+            }
+
+            Loaded = true;
+        }
+
+        public void Run()
+        {
+            if (_transformerAssembly != null)
+            {
+                var t = _transformerAssembly.GetType("EventTransformer.TransformerFactory");
+                var methodInfoStatic = t.GetMethod("GetEventTransformer");
```

Comment:
pretty misleading name. something like getEventTransformer would be more readable.